packages/cli/templates/default-app/package.json.hbs

```diff
@@ -3,9 +3,11 @@
   "private": true,
   "scripts": {
     "start": "yarn build && yarn workspace app start",
-    "build": "lerna run build",
+    "build": "lerna run build --ignore=plugin-*",
+    "prebuild": "lerna run build --scope=plugin-*",
```

Comment:
This builds plugins before the rest of the packages, which solves a problem where `app` was being built before a plugin, which in turn caused the build to fail. Can be removed when/if we don't require plugins to be built before they can be used.

Reply:
The app isn't supposed to have a build script at all actually, so I think we can keep it as is and add a bundle script like we have in this repo.

Reply:
Does it get called automatically?

Reply:
@freben Yes, it's a `pre` hook. Apparently one can add `pre` to any script and it will run before the "normal" one. There's also `post` hooks that run after...
@Rugvip 👍